theano/scalar/basic.py

```diff
@@ -851,6 +851,8 @@ class ScalarOp(Op):
     nin = -1
     nout = 1
 
+    __props__ = ("output_types_preference", "name")
```

Comment:
The class InRange override the \__init__. So it have different props. Check if there is other subclass that override init